static/js/biomedical/protein/chart.ts

```diff
@@ -968,15 +969,16 @@ export function drawVarGeneAssocChart(
     .attr("cy", (d) => y(d.id))
     .attr("r", "6")
     .attr("id", (d, i) => circleIdFunc(i))
-    .style("fill", (d) => ERROR_BAR_VAR_COLOR[d.name])
+    .style("fill", (d: VarGeneDataPoint) => ERROR_BAR_VAR_COLOR[d.name])
     // variant redirect
-    .on("click", (d) => {
+    .on("click", (d: VarGeneDataPoint) => {
```

Comment:
I see you updated a lot of the other `.on(<event>, () => {})` calls by adding an extra `event` param in the callback function, do you need to do that here as well?

Reply:
Good catch. Fixed here and a few other places.